gcloud/credentials.py

```diff
@@ -207,7 +218,11 @@ def _get_signature_bytes(credentials, string_to_sign):
         pkey = _get_pem_key(credentials)
         if not isinstance(string_to_sign, six.binary_type):
             string_to_sign = string_to_sign.encode('utf-8')
-        return crypto.sign(pkey, string_to_sign, 'SHA256')
+        if crypto is None:
```

Comment:
This check might not actually be necessary as `_get_pem_key` does the same thing and is called at 218. Though, I like the consistency. Your call.